packages/salesforcedx-apex-replay-debugger/test/unit/adapter/apexReplayDebug.test.ts

```diff
@@ -24,10 +25,9 @@ import {
   Thread
 } from '@vscode/debugadapter';
 import { DebugProtocol } from '@vscode/debugprotocol';
```

Comment:
I don't like mixing test frameworks.  Especially having both jest.mock and sinon seemed ugly.

So they're all jest now in replay-debugger